pointers/tests/pointers_tests.c

```diff
@@ -54,11 +54,11 @@ char *test_find_char()
     char *str = "LambdaSchool";
     char *found = find_char(str, 'b');
 
-    mu_assert(strcmp(found, "bdaSchool") == 0, "Your find_char function did not work as expected.");
+    mu_assert(strcmp(found, "bdaSchool") == 0, "Your find_char function did not work as expected (0).");
 
     found = find_char(str, 'S');
-
-    mu_assert(strcmp(found, "School") == 0, "Your find_char function did not work as expected.");
+  
+    mu_assert(strcmp(found, "School") == 0, "Your find_char function did not work as expected (1).");
```

Comment:
I see  you've made changes in the tests, let me know if you're having any issues regarding this